docker-compose.pullpreview.yml

```diff
@@ -23,12 +21,25 @@ worker: &ruby
     - "RAILS_ENV=production"
     - "HEROKU=true"
     - "OP_ADMIN_USER_SEEDER_FORCE_PASSWORD_CHANGE=off"
-  command: "bundle exec rake db:migrate db:seed jobs:work"
-  memory: 384
-
-web:
-  <<: *ruby
   ports:
     - "80:8080"
   command: "./docker/web"
   memory: 1024
+  volumes:
+    - .:/files
```

Comment:
@wielinde Just keep `.:/usr/src/app` here instead of `.:/files`, and then both `worker` and `web` processes will share the same `/usr/src/app` directory (so if you have `/usr/src/app/files` in the `web` container instance, it will be exactly the same shared volume in the `worker` container instance) !